students/AndyKwok/lesson03/list_lab.py

```diff
@@ -0,0 +1,102 @@
+# Description: 
+# Author: Andy Kwok
+# Last Updated: 07/14/2018
+# ChangeLog: 
+# v1.0 - Initialization
+
+import os 
+
+#!/usr/bin/env python
+
+list = ['Apples', 'Pears', 'Oranges', 'Peaches']
+
+#Series 1
+def series_one(list):
+    """ Function to display and add item to list
+    list: list of fruit
+    new_list = list after mod in fruit
+    new_fruit: new element to add to list
+    id: index value
+    new_fruit_2: another new element to add to list
+    selection: list item scan
+    """ 
+    print(list)
+    new_fruit = input('Please enter another fruit > ')
+    list = list + [new_fruit]
```

Comment:
It's best not to use Python built-ins like `list` for variable names as this will overwrite them